helm/ffc-demo-web/templates/ingress.yaml

```diff
@@ -16,21 +16,21 @@ metadata:
     nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
     nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
     nginx.ingress.kubernetes.io/configuration-snippet: |
-      proxy_set_header l5d-dst-override {{.Values.name }}.default.svc.cluster.local:{{ .Values.service.port }};
+      proxy_set_header l5d-dst-override {{quote .Values.name }}.default.svc.cluster.local:{{ quote .Values.service.port }};
       proxy_hide_header l5d-remote-ip;
       proxy_hide_header l5d-server-id;
-  name: {{ .Values.name }}
+  name: {{ quote .Values.name }}
 spec:
   rules:
 {{- if .Values.ingress.endpoint }}
-  - host: {{ .Values.ingress.endpoint }}.{{.Values.ingress.server}}
+  - host: {{ quote .Values.ingress.endpoint }}.{{quote .Values.ingress.server}}
```

Comment:
This looks incorrect. I'm not sure this file is actually used anymore, but this will result in quotes embedded in the final string.